### DISCUSSION TITLE
Cannot declare class Symfony\\Component\\Dotenv\\Dotenv

### DISCUSSION THREAD

**Discussion by annemiekebuijs** [api-platform/core#4713] [Q&A]
I have build an api with apiplatform and symfony4.
This has been up and running for almost 2 years now without any problems.
There have been no updates to anything, not to apache, not to the code, but this morning, out of the blue i see this error appearing in the `/etc/httpd/logs/<project>/error.log`

`PHP Fatal error:  Cannot declare class Symfony\\Component\\Dotenv\\Dotenv, because the name is already in use in /var/www/html/<website>/vendor/symfony/dotenv/Dotenv.php on line 26`

After restarting apache everything was alright again, but i realy want to know why this happened.

This is composer.json:
```
 "require": {
    "php": "^7.2",
    "ext-ctype": "*",
    "ext-iconv": "*",
    "ext-json": "*",
    "ext-pdo": "*",
    "api-platform/api-pack": "1.2.0",
    "beberlei/doctrineextensions": "^1.3",
    "doctrine/doctrine-bundle": "^1.6",
    "doctrine/doctrine-migrations-bundle": "^2.0",
    "gesdinet/jwt-refresh-token-bundle": "^0.9.1",
    "lexik/jwt-authentication-bundle": "^2.6",
    "nesbot/carbon": "^2.27",
    "symfony/apache-pack": "^1.0",
    "symfony/console": "4.3.*",
    "symfony/dotenv": "4.3.*",
    "symfony/flex": "^1.1",
    "symfony/framework-bundle": "4.3.*",
    "symfony/monolog-bundle": "^3.5",
    "symfony/yaml": "4.3.*"
  }
```

**Comment by soyuka** [comment]
It doesn't look API Platform related
